From 41b54fe290c5f787a77d7f891362d4299152af0b Mon Sep 17 00:00:00 2001
Message-ID: <41b54fe290c5f787a77d7f891362d4299152af0b.1731584860.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1731584860.git.sharlatanus@gmail.com>
References: <cover.1731584860.git.sharlatanus@gmail.com>
From: Ricardo Wurmus <rekado@elephly.net>
Date: Wed, 13 Nov 2024 18:16:16 +0100
Subject: [PATCH v2 08/14] gnu: python-boto3: Update to 1.35.59.

* gnu/packages/python-xyz.scm (python-boto3): Update to 1.35.59.
[native-inputs]: Remove python-nose.

Change-Id: Iafc22bafbf6ef185c2d9f4907386bd904ebaecbd
---
 gnu/packages/python-xyz.scm | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/gnu/packages/python-xyz.scm b/gnu/packages/python-xyz.scm
index da945e12fe..ca021c2776 100644
--- a/gnu/packages/python-xyz.scm
+++ b/gnu/packages/python-xyz.scm
@@ -19362,7 +19362,7 @@ (define-public python-botocore
 (define-public python-boto3
   (package
     (name "python-boto3")
-    (version "1.35.54")
+    (version "1.35.59")
     (source
      (origin
        (method git-fetch)               ; no tests in PyPI release
@@ -19371,17 +19371,16 @@ (define-public python-boto3
              (commit version)))
        (file-name (git-file-name name version))
        (sha256
-        (base32 "0w0k719i8pyzkvd2snv6z91r57k6cxkq1i3phygdpwzk0k6acvj2"))))
+        (base32 "10bdzdaw7qg2m5n5ivb2zzsdl7wgjmz05xyxajd4cmk629ick95m"))))
     (build-system pyproject-build-system)
     (arguments
      (list
       #:test-flags
       #~(list "--numprocesses" "auto"
-              ;; Tests require networking.
+              ;; Integration tests are trying to connect to AWS.
               "--ignore" "tests/integration")))
     (native-inputs
      (list python-mock
-           python-nose
            python-pytest
            python-pytest-xdist
            python-setuptools
-- 
2.46.0

